content/docs/hooks-intro.md

```diff
@@ -25,86 +25,87 @@ function Example() {
 
-We'll discuss this more in [Building Your Own Hooks](/docs/hooks-custom.html).
+נדון על כך יותר ב[בניית Hooks משלך](/docs/hooks-custom.html).
 
-### Complex components become hard to understand {#complex-components-become-hard-to-understand}
+### קומפוננטות מורכבות הופכות קשות להבנה {#complex-components-become-hard-to-understand}
 
-We've often had to maintain components that started out simple but grew into an unmanageable mess of stateful logic and side effects. Each lifecycle method often contains a mix of unrelated logic. For example, components might perform some data fetching in `componentDidMount` and `componentDidUpdate`. However, the same `componentDidMount` method might also contain some unrelated logic that sets up event listeners, with cleanup performed in `componentWillUnmount`. Mutually related code that changes together gets split apart, but completely unrelated code ends up combined in a single method. This makes it too easy to introduce bugs and inconsistencies.
+לעיתים קרובות נאלצנו לתחזק קומפוננטות שהתחילו כפשוטות וגדלו לבלאגן בלתי נשלט של לוגיקה ו-side effects. כל method ב-lifecycle לעיתים קרובות הכילה ערבוב של לוגיקה לא קשורה. לדוגמא, קומפוננטות עשויות לבצע data fetching ב`componentDidMount`  וב-`componentDidUpdate`. למרות זאת `componentDidMount` עשויה להכיל גם לוגיקה לא קשורה אשר מגדירה event listeners, עם ביצועי ניקוי ב-`componentWillUnmount`. קוד שקשור אחד לשני מתפצל, אבל קוד שלחלוטין איננו קשור מורכב בmethod אחת. זה הופך את זה לקל יותר להצגת באגים וחוסר עקביות.
 
-In many cases it's not possible to break these components into smaller ones because the stateful logic is all over the place. It's also difficult to test them. This is one of the reasons many people prefer to combine React with a separate state management library. However, that often introduces too much abstraction, requires you to jump between different files, and makes reusing components more difficult.
+במקרים רבים זה לא אפשרי להפריד קומפוננטות אילו לקטנות יותר מכיוון שהלוגיקה נמצאת בכל מקום, זה גם קשה לבדוק אותן. זאת אחת הסיבות שהרבה אנשים מעדיפים לשלב את React עם ספריית state management נפרדת. למרות זאת, לעיתים קרובות זה גם מציג יותר מידי אבסטרקציה, דורש ממך לקפוץ בין קבצים, ויוצר את השימוש החוזר בקומפוננטות קשה יותר.
 
-To solve this, **Hooks let you split one component into smaller functions based on what pieces are related (such as setting up a subscription or fetching data)**, rather than forcing a split based on lifecycle methods. You may also opt into managing the component's local state with a reducer to make it more predictable.
+כדי לפתור זאת, **Hooks נותן לנו לפצל קומפוננטה אחת לכמה פונקציות קטנות יותר בהתבסס על הקשרם (כמו להגדיר מנוי או fetching data)**, במקום להכריח פיצול המבוסס lfecycle methods.  תוכל גם לבחור בreducer לניהול ה-state המקומי של הקומפוננטה כדי להפוך אותה לצפוייה יותר.
```

Comment:
מנוי sounds off, I would keep "subscription" here

Reply:
lfecycle methods -> מתודות lifecycle
(you have a typo too)

Reply:
תוכל גם לבחור בreducer לניהול ה-state המקומי של הקומפוננטה
-> 
תוכל לבחור לנהל את ה-state המקומי של הקומפוננטה בעזרת reducer